types/db.go

```diff
@@ -67,3 +68,20 @@ type Attribute struct {
 	Key        string        `gorm:"index" json:"key"`
 	Value      string        `json:"value"`
 }
+
+// Plan is a Terraform plan
+type Plan struct {
+	ID               uint           `sql:"AUTO_INCREMENT" gorm:"primary_key" json:"id"`
+	Version          Version        `json:"version"`
+	VersionID        sql.NullInt64  `gorm:"index" json:"-"`
```

Comment:
What is the difference between the `Version` and `VersionID` fields?

Reply:
If I have understand the Gorm documentation correctly it's to make a **Belongs To** association.
You have done that to link a __Version__ to a __State__ in ```types/db.go```